src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from redminelib import Redmine
+from enum import Enum
+import errno
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+
+
+class Feedback(Enum):
+    project_id = 1,
+    tracker_id = 1,
+    subject = '',
+    description = '',
+    category_id = 195,
+    Severity = '3 - Minor'
+
+class config(Enum):
+    url = 'tracker.ceph.com'
+    port = 443
```

Comment:
Same here. This is [not an enum](https://en.wikipedia.org/wiki/Enumerated_type).

It's ok to 'namespace' things, but this is enough for that:
```
class Config:
  url = ...
  port = ...
```
Or if you want the initializer automatically created for you (and as we cannot use Py 3.7 `@dataclasses`), you can use [NamedTuples](https://docs.python.org/3/library/typing.html#typing.NamedTuple) instead.